extensions/replication/ReplicationConfigValidator.js

```diff
@@ -50,6 +50,7 @@ const joiSchema = {
     },
     topic: joi.string().required(),
     replicationStatusTopic: joi.string().required(),
+    trackReplicationFailures: joi.boolean().default(true),
```

Comment:
What do you all think of `trackReplicationFailures`? Alternatives I considered:

* `trackCRRFailures`
* `enableReplicationRetry`

Reply:
I vote  for trackCRRFailures

Reply:
`monitorReplicationFailures`?

Reply:
As an external parameter name I would prefer sticking to the service semantic, i.e. "replication" service, so `trackReplicationFailures`. Internally (function names etc.) we also have some names with CRR in it.

Reply:
`monitorReplicationFailures` is good for me too, not being an english native speaker I leave the choice to others :rofl:

Reply:
Okay, it seems like we all agree on `monitorReplicationFailures`. 💃We have a winner.

Reply:
@jonathan-gramain BTW, your English is awesome! I'm always impressed with your doc writing. 😄